namanager/core.py

```diff
@@ -74,22 +66,21 @@ def verify_setting_type(self):
              't': type(enums.SETTINGS['DIR_FORMATS']['SEP'])},
             {'s': self._DIR_LETTER_CASE,
              't': type(enums.SETTINGS['DIR_FORMATS']['LETTER_CASE'])},
-            {'s': self._ONLY_DIRS,
-             't': type(enums.SETTINGS['ONLY_DIRS'])},
+            {'s': self._INVOLVE_DIRS,
+             't': type(enums.SETTINGS['INVOLVE_DIRS'])},
             {'s': self._IGNORE_DIRS,
              't': type(enums.SETTINGS['IGNORE_DIRS'])},
         ]
 
         for setting in settings:
             if not isinstance(setting['s'], setting['t']):
                 errors.append("Type of {0} must be {1}.".format(
-                    self.name(setting['s'], self.__dict__), setting['t']))
-
-        err_str = '\n'
-        for error in errors:
-            err_str += error + '\n'
+                    util.name(setting['s'], self.__dict__), setting['t']))
 
         if errors:
+            err_str = '\n'
+            for error in errors:
```

Comment:
I think using a list comprehension would be possible here.

Reply:
I couldn't catch what you said.